src/main/java/com/exalttech/trex/ui/controllers/MainViewController.java

```diff
@@ -1416,6 +1423,21 @@ public void onPortListUpdated(boolean successfullyUpdated) {
     }
 
     /**
+     * Re-acquire owned port on login 
+     */
+    private void reaquireOwnedPorts() {
+        try{
+          for (Port port : portManager.getPortList()) {
+              if(portManager.isCurrentUserOwner(port.getIndex())){
+                  serverRPCMethods.acquireServerPort(port.getIndex(), true);
+              }
+          }
+        }catch(Exception ex){
+            LOG.error("Error re-acquiring port", ex);
```

Comment:
Use the actual exception not generic.

If multiple exceptions with the same target do (ExceptionType1 | ExceptionType2 ex)